app/controllers/users_controller.rb

```diff
@@ -1,17 +1,28 @@
 class UsersController < ApplicationController
-
+  before_action :set_user, except: [:index, :new, :create]
+  
   def index
     @users = User.all
   end
 
   def show
-    @user = User.find(params[:id])
+  end
```

Comment:
Why this method is empty?

Reply:
By the before_action

Reply:
Then delete the method, it's not necessary.